commit 3c7306589dddbcc0ac53ec6b2c3a14875879e20c
Author: Bhupesh Sharma <bhupesh.sharma@linaro.org>
Date:   Sat Jan 28 11:05:03 2023 +0530

    remoteproc: qcom: pas: Add sm6115 remoteprocs
    
    Among the subsystems in the Qualcomm sm6115 platform we find
    audio, compute and modem DSPs.
    
    Add support for controlling these using the peripheral
    authentication service (PAS) remoteproc driver.
    
    Signed-off-by: Bhupesh Sharma <bhupesh.sharma@linaro.org>
    Reviewed-by: Mukesh Ojha <quic_mojha@quicinc.com>
    Acked-by: Krzysztof Kozlowski <krzysztof.kozlowski@linaro.org>
    Signed-off-by: Bjorn Andersson <andersson@kernel.org>
    Link: https://lore.kernel.org/r/20230128053504.2099620-1-bhupesh.sharma@linaro.org

diff --git a/drivers/remoteproc/qcom_q6v5_pas.c b/drivers/remoteproc/qcom_q6v5_pas.c
index 7418f0e61943..3b735ff6713b 100644
--- a/drivers/remoteproc/qcom_q6v5_pas.c
+++ b/drivers/remoteproc/qcom_q6v5_pas.c
@@ -1202,6 +1202,9 @@ static const struct of_device_id adsp_of_match[] = {
 	{ .compatible = "qcom,sdm845-adsp-pas", .data = &sdm845_adsp_resource_init},
 	{ .compatible = "qcom,sdm845-cdsp-pas", .data = &sdm845_cdsp_resource_init},
 	{ .compatible = "qcom,sdx55-mpss-pas", .data = &sdx55_mpss_resource},
+	{ .compatible = "qcom,sm6115-adsp-pas", .data = &adsp_resource_init},
+	{ .compatible = "qcom,sm6115-cdsp-pas", .data = &cdsp_resource_init},
+	{ .compatible = "qcom,sm6115-mpss-pas", .data = &sc8180x_mpss_resource},
 	{ .compatible = "qcom,sm6350-adsp-pas", .data = &sm6350_adsp_resource},
 	{ .compatible = "qcom,sm6350-cdsp-pas", .data = &sm6350_cdsp_resource},
 	{ .compatible = "qcom,sm6350-mpss-pas", .data = &mpss_resource_init},